File: 프로그래머스/1/135808. 과일 장수/과일 장수.py
# k가 굳이 주어진 이유??
def solution(k, m, score):
    answer = 0
    score.sort(reverse=True)
    box = len(score) // m
        
    for i in range(1, box + 1):
        answer += score[i * m - 1] * m
    
    
    return answer

File: 프로그래머스/1/135808. 과일 장수/과일 장수.java
import java.util.*;

class Solution {
    public int solution(int k, int m, int[] score) {
        int answer = 0;
        
        Arrays.sort(score);
        
        for (int i = score.length-1; i > -1; i -= m) {
            if (i-m+1 > -1) {
            	answer += score[i - m + 1] * m;
            }
        }
        
        return answer;
    }
}